app/uk/gov/hmrc/thirdpartyapplication/connector/EmailConnector.scala

```diff
@@ -106,6 +109,35 @@ class EmailConnector @Inject()(httpClient: HttpClient, config: EmailConfig)(impl
       Map("applicationName" -> application, "requestor" -> requesterEmail)))
   }
 
+  def sendAddedClientSecretNotification(actorEmailAddress: String,
+                                        clientSecret: String,
+                                        applicationName: String,
+                                        recipients: Set[String])(implicit hc: HeaderCarrier): Future[HttpResponse] = {
+    sendClientSecretNotification(addedClientSecretNotification, actorEmailAddress, clientSecret, applicationName, recipients)
+  }
+
+  def sendRemovedClientSecretNotification(actorEmailAddress: String,
+                                          clientSecret: String,
+                                          applicationName: String,
+                                          recipients: Set[String])(implicit hc: HeaderCarrier): Future[HttpResponse] = {
+    sendClientSecretNotification(removedClientSecretNotification, actorEmailAddress, clientSecret, applicationName, recipients)
+  }
+
+  private def sendClientSecretNotification(templateId: String,
+                                           actorEmailAddress: String,
+                                           clientSecret: String,
+                                           applicationName: String,
+                                           recipients: Set[String])(implicit hc: HeaderCarrier): Future[HttpResponse] = {
+    post(SendEmailRequest(recipients, templateId,
+      Map(
+        "actorEmailAddress" -> actorEmailAddress,
+        "clientSecretEnding" -> clientSecret.takeRight(4), // scalastyle:off magic.number
```

Comment:
Can we use the client secret name here instead?

Reply:
(And in the future, client secret will be going away)

Reply:
Yeah good point. I've made a change here.